packages/multi-test/src/wasm.rs

```diff
@@ -61,15 +59,19 @@ where
     ) -> AnyResult<Binary>;
 
     /// Handles all WasmMsg messages
-    fn execute(
+    fn execute<BankT, CustomT>(
```

Comment:
Shouldn't we enforce that CustomT makes sense? Such that CustomT is related to ExecC and QueryC?

Reply:
Ah, I just saw the where clause. Perfect.